lib/usb_connection.js

```diff
@@ -82,22 +82,8 @@ USB.Connection.prototype._receiveMessages = function() {
   var transferSize = 4096;
   // Start polling
   self.epIn.startPoll(2, transferSize);
-
   // When we get data, push it into the stream
   self.epIn.on('data', self.push.bind(self));
-
-  // Report errors as they arise
-  self.epIn.on('error', function(e) {
```

Comment:
Removes the event listener that immediately kills the script upon endpoint error. This is bad because all process exiting should be through `tessel-2-bin.js` and because there is no way to handle it. 

However, by removing this, miscellaneous endpoint issues that occur after the `open` function aren't going to be caught. We can revisit this issue later.